docker-compose.yml

```diff
@@ -180,4 +180,38 @@ services:
       - "traefik.port=80"
       - "traefik.frontend.rule=PathPrefix: /"
       - "traefik.website.frontend.whiteList.sourceRange=${TRAEFIK_IPFILTER:-172.16.0.0/12}"
-    container_name: "frontend"
\ No newline at end of file
+    container_name: "frontend"
+
+  elasticsearch:
+    container_name: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.2
+    ports:
+      - 9200:9200
```

Comment:
in general we don't want to expose ports, for local development this can be done using docker-compose.override.yaml.